src/unit_tests/shared/test_agent_op.c

```diff
@@ -25,22 +25,22 @@
 
 /* redefinitons/wrapping */
 
-extern cJSON* w_create_agent_add_payload(const char *name, const char *ip, const char * groups, const char *key, const int force, const char *id);
+extern cJSON* w_create_agent_add_payload(const char *name, const char *ip, const char *groups, const char *key_hash, const char *key, const char *id, const int force);
 extern cJSON* w_create_agent_remove_payload(const char *id, const int purge);
 extern cJSON* w_create_sendsync_payload(const char *daemon_name, cJSON *message);
 extern int w_parse_agent_add_response(const char* buffer, char *err_response, char* id, char* key, const int json_format, const int exit_on_error);
 extern int w_parse_agent_remove_response(const char* buffer, char *err_response, const int json_format, const int exit_on_error);
 
-
 static void test_create_agent_add_payload(void **state) {
     char* agent = "agent1";
     char* ip = "192.0.0.0";
     char* groups = "Group1,Group2";
     char* key = "1234";
+    char* key_hash = "7110eda4d09e062aa5e4a390b0a572ac0d2c0220";
```

Comment:
As this is a dummy value of key_hash we should keep it easy as "**ABCD**"

Reply:
Maybe, the reason behind this is to use the parameters values as real as possible.
But if it isn't necessary, we can change it.

Reply:
It's good, then.